library/Ivoz/Provider/Domain/Service/Domain/UpdateByCompany.php

```diff
@@ -76,6 +77,19 @@ public function execute(CompanyInterface $company)
         $domain = $this->entityTools
             ->persistDto($domainDto, $domain);
 
-        $company->setDomain($domain);
```

Comment:
This a PRE_PERSIST event, I thought it was correct to use setters in LifeCycled entity here.

Reply:
Entity setters will be protected functions soon, that's why I'm replacing them here